release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -3460,3 +3460,27 @@ link:https://access.redhat.com/solutions/6663261[{product-title} 4.6.54 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.6 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-6-55"]
+=== RHBA-2022:0566 - {product-title} 4.6.55 bug fix and security update
+
+Issued: 2022-02-23
+
+{product-title} release 4.6.55, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:0566[RHBA-2022:0566] advisory. There are no RPM packages for this release. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:0565[RHSA-2022:0565] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
```

Comment:
For the second sentence, it is advisable, to start with a prefacing phrase, and then the xref/link:

"For notes about the container images in this release, see the following article:"

Also, use "about" rather than "on".

Reply:
@lpettyjo Thanks for pointing this out! I agree that we should change this. But since it's boilerplate text that is repeated a bazillion times in this and other RN assemblies, I think I'll STET this for now. It would be best to make this change all the way through in another PR.